presto-main/src/main/java/io/prestosql/server/protocol/Slug.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.server.protocol;
+
+import com.google.common.hash.HashFunction;
+import com.google.common.hash.Hashing;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static java.util.Locale.ENGLISH;
+import static java.util.Objects.requireNonNull;
+import static java.util.UUID.randomUUID;
+
+public final class Slug
+{
+    public static Slug createNew()
+    {
+        return fromKey(randomUUID().toString().toLowerCase(ENGLISH).replace("-", ""));
```

Comment:
Instead of going through UUID to get 128bits of random, just use:
```java
byte[] randomBytes = new byte[16];
ThreadLocalRandom.current().nextBytes(randomBytes);
return  new Slug(randomBytes);
```

Reply:
It's just moved from `io.prestosql.dispatcher.QueuedStatementResource.Query#slug` initialized, so I will change it in a separate commit.

Reply:
I will go with `SecureRandom` though.